bin/_test-run.sh

```diff
@@ -191,45 +195,48 @@ install_stable() {
 # this branch.
 # $1 - namespace to use for the stable release
 run_upgrade_test() {
-    local stable_namespace=$1
-    local stable_version=$(latest_stable)
+    local stable_namespace
+    local stable_version
+    stable_namespace=$1
+    stable_version=$(latest_stable)
```

Comment:
These still should be fine as the below I think?

```
local stable_namespace=$1
local stable_version=$(latest_stable)
```
Any reason for splitting them out?

Reply:
Yes, it is due to SC2155, "Declare and assign separately to avoid masking return values" with the details explained here:

https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
I only checked `stable_namespace` which is okay as `local stable_namespace=$1`.

Makes sense that `stable_version` which is being assigned to a return value though would have the lint though.